Tests/SentryTests/PrivateSentrySDKOnlyTests.swift

```diff
@@ -3,6 +3,21 @@ import XCTest
 
 class PrivateSentrySDKOnlyTests: XCTestCase {
 
+    private class Fixture {
+        static func getUnhandledExceptionEnvelope() -> SentryEnvelope {
```

Comment:
`m`: We're not using the `Fixtures` pattern anymore, because it's so verbose. I would just add `getUnhandledExceptionEnvelope` as a method to `PrivateSentrySDKOnlyTests` without creating the extra class.